### PR TITLE
Add Flipsnack

### PR DESCRIPTION
**Pull request by Yang-Nankai**
Flipsnack is an online platform for creating, sharing, and publishing digital flipbooks, brochures, magazines, and catalogs. It allows users to design interactive publications from scratch or convert PDF files into flipbooks with customizable features and layouts. Flipsnack provides 2FA for the account login verification process. The website is ranked 23,533 on [Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=flipsnack.com). The website provides documentation on how to turn on 2FA.

### CODE REVIEW

File: img/f/flipsnack.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 8 85 65"><path fill="#0362FC" d="M75.8 66.4a59 59 0 0 1-13.3 2c3.8-6.1 14.2-30 19-45.3v33.7c0 4.1-1 8.2-5.7 9.6zm-33 .4c.3-7.8-.2-27.8-.6-38C41.5 8.2 68 18 82 16.8v.3a157.9 157.9 0 0 1-31.6 38.8c10.8 7.8 11.2 12.6 6.8 12.6h-.3c-5-.3-9.7-.7-14-1.7zM41.5 16a69.4 69.4 0 0 0-39.8-5.2c-3.1 15.5-1.4 36 1 51.8 1.2 7.8 1.8 8.9 7.5 7.1 22.2-6.1 41.8 6.2 63.9.7 7.4-1.8 10.2-5.5 10.5-12.3l1.1-45.3c-11.3.4-36.8-4-44.2 3.2z"/></svg>
```
Comment (fletcherdodd) [inline img/f/flipsnack.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 85.6 62"><path fill="#0362fc" d="M75.7 56.5a59.2 59.2 0 0 1-13.3 2 300 300 0 0 0 19-45.3V47c0 4.1-1 8.2-5.7 9.6m-33 .4c.3-7.8-.2-27.8-.6-38C41.4-1.7 67.9 8.1 81.9 7v.3A157.8 157.8 0 0 1 50.3 46C61 53.8 61.5 58.6 57 58.6h-.3c-5-.3-9.7-.7-14-1.7zM41.4 6.1A69.3 69.3 0 0 0 1.6 1a164 164 0 0 0 1 51.8c1.2 7.8 1.8 9 7.5 7.1 22.2-6 41.8 6.2 63.9.7 7.4-1.8 10.2-5.5 10.5-12.3l1-45.3c-11.2.4-36.7-4-44.1 3.2"/></svg>
```

**Comment by Yang-Nankai** [comment]
I would like to ask what tool you use to modify SVG? I use the SVG plugin for VSCode, but I feel that I can never control the size of the SVG image (`viewBox`) very well. @fletcherdodd

**Comment by fletcherdodd** [comment]
> I would like to ask what tool you use to modify SVG? I use the SVG plugin for VSCode, but I feel that I can never control the size of the SVG image (`viewBox`) very well. @fletcherdodd 

I use a combination of a few things;

**Illustrator -** I use this to tidy up the SVG file and export it so there are no gaps in the image file. _If you don't have program that can edit an SVG there are a few ok ones online._
**[Vector.io](https://vecta.io/nano) -** I then put whatever I have exported from Illustrator through this tool. This removes a lot of unnecessary stuff that Illustrator adds (data-name elements, etc.) and usually compresses it quite a bit.
**[SVG Viewer](https://www.svgviewer.dev) -** then I upload the file downloaded from Vector.io into this and make sure it hasn't messed it up. I'll removed any additional unnecessary elements it may have missed.
**[SVGOMG](https://jakearchibald.github.io/svgomg/) -** Lastly, I'll copy the code from SVG Viewer and use the number precision slider on the right-side panel to see if I can optimise it even more. I slide it to as small as I can without ruining the logo/icon. Sometimes it's already as optimised as it can be. I may copy the code back to SVG Viewer just to make sure things look ok in there too before added it to my PR.

A bit of work I know, but I have found using a combination of all of these tools yields the best results

**Review by fletcherdodd — APPROVED** [review]

**Comment by Yang-Nankai** [comment]
Thank you for your patience in answering! I will try to try these tools and processes! ❤️

**Pull request merged**